crates/subspace-farmer/src/bin/subspace-farmer/commands/bench.rs

```diff
@@ -220,10 +247,13 @@ pub(crate) async fn bench(
         HumanReadable(overhead),
         (overhead * 100) as f64 / space_allocated as f64
     );
-    println!("{:.2?} plotting time", took);
+    println!("{} plotting time", HumanReadable(took));
     println!(
         "{:.2}M/s average plotting throughput",
-        write_pieces_size as f64 / 1000. / 1000. / took.as_secs_f64()
+        (write_pieces_size * multi_farming.plots.len() as u64) as f64
```

Comment:
We've plotted everything in `took` amount of time, why do you multiply it by number of plots?

BTW `tokio::spawn` above is unnecessary unless I'm missing something.